snack-lib/wrapper.nix

```diff
@@ -0,0 +1,14 @@
+{ snackNix
+, nixpkgs ? null
+}:
+  let
+    pkgs =
+      if nixpkgs == null
+      then import <nixpkgs> {}
+      else import nixpkgs {};
```

Comment:
what if I want to configure my nixpkgs?

Reply:
then you pass your own `nixpkgs` that's preconfigured

Reply:
well it's initialized by passing `{}`